src/services/grafana-provisioning/dashboards/10619_rev2.json

```diff
@@ -0,0 +1,1278 @@
+        "hide": 0,
+        "includeAll": false,
+        "label": "Port",
+        "multi": false,
+        "name": "port",
+        "options": [],
+        "query": "label_values(container_cpu_user_seconds_total{instance=~\"$node:(.*)\"}, instance)",
+        "refresh": 1,
+        "regex": "/[^:]+:(.*)/",
+        "skipUrlSync": false,
+        "sort": 3,
+        "tagValuesQuery": "",
+        "tagsQuery": "",
+        "type": "query",
+        "useTags": false
+      },
+      {
+        "allValue": "All",
+        "current": {
+          "selected": false,
+          "text": [
+            "All"
+          ],
+          "value": [
+            "$__all"
+          ]
+        },
+        "datasource": {
+          "type": "prometheus",
+          "uid": "prometheusdatasource"
+        },
+        "definition": "label_values(container_cpu_user_seconds_total{instance=~\"10.115.220.32:8080|10.115.220.33:8080\", name!=\"cadvisor\"}, name)",
+        "hide": 0,
+        "includeAll": true,
+        "label": "Prod",
+        "multi": true,
+        "name": "Prod",
+        "options": [],
+        "query": "label_values(container_cpu_user_seconds_total{instance=~\"10.115.220.32:8080|10.115.220.33:8080\", name!=\"cadvisor\"}, name)",
+        "refresh": 1,
+        "regex": "",
+        "skipUrlSync": false,
+        "sort": 1,
+        "tagsQuery": "",
+        "type": "query",
+        "useTags": false
+      },
+      {
+        "allValue": "All",
+        "current": {
+          "selected": false,
+          "text": [
+            "All"
+          ],
+          "value": [
+            "$__all"
+          ]
+        },
+        "datasource": {
+          "type": "prometheus",
+          "uid": "prometheusdatasource"
+        },
+        "definition": "label_values(container_cpu_user_seconds_total{instance=~\"10.115.222.21:8080|10.115.222.22:8080\", name!=\"cadvisor\"}, name)",
+        "hide": 0,
+        "includeAll": true,
+        "label": "NonProd",
+        "multi": true,
+        "name": "NonProd",
+        "options": [],
+        "query": "label_values(container_cpu_user_seconds_total{instance=~\"10.115.222.21:8080|10.115.222.22:8080\", name!=\"cadvisor\"}, name)",
+        "refresh": 1,
+        "regex": "",
+        "skipUrlSync": false,
+        "sort": 0,
+        "tagValuesQuery": "",
+        "tagsQuery": "",
+        "type": "query",
+        "useTags": false
+      },
+      {
+        "current": {
+          "selected": true,
+          "text": [
+            "warnet_tank_000006",
+            "warnet_tank_000008",
+            "warnet_tank_000001",
+            "warnet_tank_000002",
+            "warnet_tank_000003",
+            "warnet_tank_000010",
+            "warnet_tank_000009",
+            "warnet_tank_000011",
+            "warnet_tank_000000",
+            "warnet_tank_000005",
+            "warnet_tank_000004",
+            "warnet_tank_000007"
```

Comment:
hard-coded tank names? This might work on network=warnet up to 12 tanks but what about other network names / number of tanks? Running default network (which should match this list) I don't see anything in the dashboard:
<img width="266" alt="Screen Shot 2023-11-01 at 5 37 02 PM" src="https://github.com/bitcoin-dev-project/warnet/assets/2084648/c7081bb8-6920-405e-9792-d2d226dcca99">

Reply:
Damn, I didn't know that would be in there. It's supposed to use a formula!?

Reply:
Fixed now I think, and added an "all" (which is the default on).

FWIW you do need to wait a minute or so for some graphs to move off "no data"...